hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Constants.java

```diff
@@ -1309,4 +1309,17 @@ private Constants() {
    * The bucket region header.
    */
   public static final String BUCKET_REGION_HEADER = "x-amz-bucket-region";
+
+  /**
+   * Is the higher performance copy from local file to S3 enabled?
+   * This switch allows for it to be disabled if there are problems.
+   * Value: {@value}.
+   */
+  public static final String COPY_FROM_LOCAL_ENABLED = "fs.s3a.copy.from.local.enabled";
```

Comment:
should we add something like "performance" or "fast" to differentiate this from normal copyFromLocal op?

Reply:
i see. maybe "optimized"